doc/faq/virtualenv_faq.rst

```diff
@@ -0,0 +1,106 @@
+.. _virtualenv-faq:
+
+***********************************************
+Working with Matplotlib in Virtual environments
+***********************************************
+
+.. contents::
+   :backlinks: none
+
+
+.. _introduction:
+
+Introduction
+============
+
+When running :mod:`matplotlib` in a virtual environment you may discover a
```

Comment:
We should probably start with links to what we're talking about.

https://virtualenv.pypa.io/en/latest/

or, in Python 3.3 and later:

https://docs.python.org/3/library/venv.html